features/0211-route-coordination/README.md

```diff
@@ -39,10 +45,27 @@ This message serves as a request from the recipient to the router, asking for th
 ```jsonc
 {
     "@id": "123456781",
-    "@type": "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/routecoordination/1.0/route-request"
+    "@type": "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/routecoordination/1.0/route-request",
+    "terms": [],
```

Comment:
The distinction "terms" vs. "user terms" feels client-server-ish and centralized to me. How about "mediator_terms" and "mediated_terms" instead?

Reply:
Or alternately, "router_terms" and "routed_terms"?

Reply:
(To me "routing" includes both mediation and relay, but terms of service make a lot more sense for mediators. This is because neither the sender nor the recipient is guaranteed to know about relays -- but both have to know about mediators. Mediators decrypt something, and under the enhanced forwarding protocol that I might eventually raise as an RFC proposal, they do more than just push a message along. They might deliver it after a delay, hold it until it's retrieved, etc.)

Reply:
`router_terms` and 'recipient_terms`? This would match the language elsewhere in this RFC.

Reply:
It seems to me that you're trying to shift terminology, popularizing "router" instead of "mediator". What I'm trying to point out is that routing is bigger than mediation, but the concepts you're talking about are pure mediation. So I'm still feeling dissonance.